lib/cfg.c

```diff
@@ -555,10 +554,10 @@ _load_file_into_string(const gchar *fname)
                 evt_tag_str(EVT_TAG_OSERROR, error->message));
 
       g_error_free(error);
-      g_string_free(content, TRUE);
       return NULL;
     }
 
+  GString *content = g_string_new("");
   g_string_append(content, buff);
```

Comment:
You can create the GString right from buff.

```
 GString *content = g_string_new(buff);
```

Reply:
done.